### ISSUE TITLE
html_safe in views is unsafe?

### ISSUE THREAD

**Issue by takah** [sds/slim-lint#39]
slim-lint v0.8.0 with rubocop v 0.41.1 reports that html_safe in slim is unsafe. Is this true?

```
app/views/events/show.html.slim:65 [W] RuboCop: Rails/OutputSafety: Tagging a string as html safe may be a security risk, \
          prefer `safe_join` or other Rails tag helpers instead
```

This blog
https://makandracards.com/makandra/2579-everything-you-know-about-html_safe-is-wrong
says that html_safe in views is safe.

**Comment by sds** [comment]
Hey @takah,

The answer here is the classic "it depends." This is not the domain of `slim-lint` however&mdash;that error is reported by `rubocop`, so if you'd like to understand the reasoning behind the warning it's best to open an issue with that project.

If you find the error noisy, you can always disable it by configuring the `ignored_cops` option of the `RuboCop` linter or disabling the cop in your `.rubocop.yml`. 

Hope that helps!
